### PR TITLE
v9.9 changelog

### PR DESCRIPTION
**Pull request by amyblais**

### CODE REVIEW

**Review by nickmisasi — APPROVED** [review]

**Review by cpoile — APPROVED** [review]

**Review by nab-77 — CHANGES_REQUESTED** [review]

**Review by agnivade — APPROVED** [review]

File: source/deploy/mattermost-changelog.md [hunk 1]
```diff
@@ -11,6 +11,104 @@ From Mattermost v9.2, this changelog summarizes updates for the latest cloud and
...
+ - Fixed an issue with the status management to avoid missing notifications.
+ - Fixed an issue where audit events were not added for OAuth logins.
+ - Fixed an issue with the error check in the message export process.
+ - Fixed an issue where pasting in the post text box could not always paste without formatting.
+ - Fixed some plugin settings with defaults not changing value.
+
+### config.json
+New setting option were added to ``config.json``. Below is a list of the additions and their default values on install. The settings can be modified in ``config.json``, or the System Console when available.
+
+#### Changes to all plans:
```
[the remark this answers is not in the document]
Comment (marianunez) [reply source/deploy/mattermost-changelog.md:80]: Correct, we can move this under the Enterprise changes plan

[the remark this answers is not in the document]
Comment (marianunez) [reply source/deploy/mattermost-changelog.md:80]: @nab-77 Also we should probably have a specific entry in the Administration or Performance section to mention the new metrics that are now available in this release.

File: source/deploy/mattermost-changelog.md [hunk 2]
```diff
@@ -11,6 +11,105 @@ From Mattermost v9.2, this changelog summarizes updates for the latest cloud and
...
+ - Fixed an issue where the user status would incorrectly be set to offline without checking for connections in other nodes in an High Availability cluster.
+ - Fixed a longstanding issue where the at-mention autocomplete could erase post text following the auto-completed at-mention.
+ - Fixed an issue with the status management to avoid missing notifications.
+ - Fixed an issue where audit events were not added for OAuth logins.
+ - Fixed an issue with the error check in the message export process.
+ - Fixed an issue where pasting in the post text box could not always paste without formatting.
+ - Fixed some plugin settings with defaults not changing value.
+
+### config.json
+New setting option were added to ``config.json``. Below is a list of the additions and their default values on install. The settings can be modified in ``config.json``, or the System Console when available.
```
Comment (wiggin77) [inline source/deploy/mattermost-changelog.md:80]:
```suggestion
New setting options were added to ``config.json``. Below is a list of the additions and their default values on install. The settings can be modified in ``config.json``, or the System Console when available.
```

File: source/deploy/mattermost-changelog.md [hunk 3]
```diff
@@ -11,6 +11,105 @@ From Mattermost v9.2, this changelog summarizes updates for the latest cloud and
...
+#### Performance
+ - Added the initial version of new client-side performance metrics.
+ - Added a metric to track time it takes for the right-hand side to load.
+ - Improved js memory profile of status’s reducers.
+
+### Bug Fixes
+ - Fixed an issue with ``aria-label`` for sidebar channel buttons.
+ - Fixed an issue where any remaining unclosed database RPC connections were not closed after a plugin shut down.
+ - Fixed an issue where the right-hand side stole the focus when coming back from threads or drafts.
+ - Fixed an issue where the clipboard did not receive the markdown as was entered into the original post instead of one that included a proxy link when clicking on the "Copy Text" list item in a channel post dot menu (accessed by clicking the three dots) for a post that includes embedded images using markdown.
```
Comment (wiggin77) [inline source/deploy/mattermost-changelog.md:66]: I find this sentence hard to parse.

**Review by wiggin77 — APPROVED** [review]
LGTM :+1:

**Review by streamer45 — APPROVED** [review]

**Review by cwarnermm — COMMENTED** [review]
Provided editorial feedback inline

File: source/deploy/mattermost-changelog.md [hunk 4]
```diff
@@ -11,6 +11,106 @@ From Mattermost v9.2, this changelog summarizes updates for the latest cloud and
...
+ - Pre-packaged GitLab plugin version [v1.9.0](https://github.com/mattermost/mattermost-plugin-gitlab/releases/tag/v1.9.0).
+ - Pre-packaged Zoom plugin version [v1.8.0](https://github.com/mattermost/mattermost-plugin-zoom/releases/tag/v1.8.0).
+ - Updated the default themes to reduce eye strain (particularly on the dark themes).
+ - Added UI improvements to the core layout.
+ - Improved error text when inviting guests to a team.
+ - Increased the visibility of the used edition of Mattermost in the interface when you are using Free edition.
+ - Added an **Unsupported** label to the Team/E0 editions in the product menu.
+ - Improved the look and feel of the **True/False** selector in the System Console.
+ - Updated the channel header layout to reduce height and simplify the UI.
+ - Fetching of complete root posts and threads of incoming thread messages are no longer allowed if they are from another channel.
```
[the remark this answers is not in the document]
Comment (amyblais) [reply source/deploy/mattermost-changelog.md:39]: @M-ZubairAhmed Would you know? cc @marianunez 
Ticket: https://mattermost.atlassian.net/browse/MM-57745

[the remark this answers is not in the document]
Comment (amyblais) [reply source/deploy/mattermost-changelog.md]: Thank you! I also moved it to the "Performance" section.

**Review by M-ZubairAhmed — APPROVED** [review]

**Review by cwarnermm — APPROVED** [review]

File: source/deploy/mattermost-changelog.md [hunk 5]
```diff
@@ -11,6 +11,106 @@ From Mattermost v9.2, this changelog summarizes updates for the latest cloud and
...
+ - Changed the semantics of the ``mattermost_http_websockets_total`` metric to track all open WebSocket connections, regardless of whether they are authenticated.
+ - Added a ``origin_client`` label to the ``mattermost_http_websockets_total`` Prometheus metric.
+
+### Go Version
+ - v9.9 is built with Go ``v1.21.8``.
+
+### Open Source Components
+ - Removed ``@stripe/react-stripe-js`` and ``@stripe/stripe-js``, and added ``web-vitals`` at https://github.com/mattermost/mattermost.
+
+### Known Issues
```
Comment (marianunez) [inline source/deploy/mattermost-changelog.md:104]: @amyblais can we add a know issue for https://mattermost.atlassian.net/browse/MM-58500 that is fixed in v9.9.1 that affects the notifications metrics delivery rate: 

```
- Inactive websockets were being counted as part of notifications delivery rate for web and desktop that caused an incorrect decrease in that metric in the Notifications Health dashboard. 
```

cc/ @devinbinnie does the above looks good to you?

File: source/deploy/mattermost-changelog.md [hunk 6]
```diff
@@ -11,6 +11,106 @@ From Mattermost v9.2, this changelog summarizes updates for the latest cloud and
...
+ - Made LDAP sync jobs more resilient against errors.
+ - Removed the ``PostPriority`` feature flag.
+ - Improved the error message of ``NotFound`` errors in store.
+ - Added support for post priority to incoming webhooks and outgoing webhook responses.
+ - Added a validation that the payload for an open Interactive Dialog request is valid according to the rules at https://developers.mattermost.com/integrate/plugins/interactive-dialogs/.
+ - Unblocked notification calls by using usernames instead of full names in case of a missing user profile.
+ - Increased the maximum password limit from 64 to 72 characters (``PasswordMaximumLength``).
+
+#### Performance
+ - Added the initial version of new client-side performance metrics.
```
Comment (marianunez) [inline source/deploy/mattermost-changelog.md:58]:
```suggestion
 - Added the initial version of new client-side performance metrics to track web app performance and can be monitored in [new Grafana board](link TBD) 
```

File: source/deploy/mattermost-changelog.md (hunk 5 above)
Comment (marianunez) [reply source/deploy/mattermost-changelog.md:104]: @amyblais  I would also like to add a link to the dashboard here as well when available in Grafana.com. We'll work on it this week so we have it before publishing the release notes

**Review by gabrieljackson — APPROVED** [review]

File: source/deploy/mattermost-changelog.md [hunk 7]
```diff
@@ -11,6 +11,108 @@ From Mattermost v9.2, this changelog summarizes updates for the latest cloud and
...
+ - Added a ``origin_client`` label to the ``mattermost_http_websockets_total`` Prometheus metric.
+
+### Go Version
+ - v9.9 is built with Go ``v1.21.8``.
+
+### Open Source Components
+ - Removed ``@stripe/react-stripe-js`` and ``@stripe/stripe-js``, and added ``web-vitals`` at https://github.com/mattermost/mattermost.
+
+### Known Issues
+ - Inactive websockets are counted as part of notifications delivery rate for web and desktop that cause an incorrect decrease in that metric in the Notifications Health dashboard [MM-58500](https://mattermost.atlassian.net/browse/MM-58500).
```
Comment (marianunez) [inline source/deploy/mattermost-changelog.md:105]:
```suggestion
 - Inactive websockets are counted as part of notifications delivery rate for web and desktop that cause an incorrect decrease in that metric in the [Notifications Health dashboard](https://grafana.com/grafana/dashboards/21305) [MM-58500](https://mattermost.atlassian.net/browse/MM-58500).
```

File: source/deploy/mattermost-changelog.md [hunk 8]
```diff
@@ -11,6 +11,108 @@ From Mattermost v9.2, this changelog summarizes updates for the latest cloud and
...
+ - Made LDAP sync jobs more resilient against errors.
+ - Removed the ``PostPriority`` feature flag.
+ - Improved the error message of ``NotFound`` errors in store.
+ - Added support for post priority to incoming webhooks and outgoing webhook responses.
+ - Added a validation that the payload for an open Interactive Dialog request is valid according to the rules at https://developers.mattermost.com/integrate/plugins/interactive-dialogs/.
+ - Unblocked notification calls by using usernames instead of full names in case of a missing user profile.
+ - Increased the maximum password limit from 64 to 72 characters (``PasswordMaximumLength``).
+
+#### Performance
+ - Added the initial version of new client-side performance metrics to track web app performance and can be monitored in new [Grafana board](https://grafana.com/grafana/dashboards/21305).
```
Comment (marianunez) [inline source/deploy/mattermost-changelog.md:58]: @amyblais this will be a different Grafana board than the metrics one. We are still waiting on the link for this one. @hmhealey is working on uploading this to Grafana.com

Comment (amyblais) [reply source/deploy/mattermost-changelog.md]: @marianunez I'm merging this changelog to the release branch in prep for tomorrow's release day https://github.com/mattermost/docs/pull/7137. Ping me when the dashboard link is ready.

**Comment by amyblais** [comment]
I'm merging this changelog to the release branch in prep for tomorrow's release day https://github.com/mattermost/docs/pull/7137.
